src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -2,46 +2,44 @@
 
-This means that patches including migrations should only be pushed from the quarantine repository to `gecko-strings` following the steps described in this document.
+This means that patches that include migrations should only be pushed from the quarantine repository to `main` following the steps described in this document.
 
 ## Running Fluent migrations
 
-This section of the document describes in detail which operations need to be performed in order to run migrations on over 150 Mercurial repositories for Firefox.
+This section of the document describes in detail which operations need to be performed in order to run migrations.
 
 ### Prerequisites
 
 Several access rights are required:
-* Write access to [l10n-central](https://hg.mozilla.org/l10n-central), i.e. SSH access with `active_scm_l10n` rights. It’s possible to check your *Access Groups* in [your profile on people.mozilla.org](https://people.mozilla.org/e#nav-access-groups).
+* Admin access to `firefox-l10n-source` and `firefox-l10n` (as a member of the `l10n-firefox` team).
 * Admin access to [Pontoon](https://pontoon.mozilla.org/).
 * An account on Heroku with access to [Pontoon apps](https://dashboard.heroku.com/apps/mozilla-pontoon/resources).
 
 In order to run migrations:
 * Install the [fluent.migrate](https://pypi.org/project/fluent.migrate/) Python package.
-* Clone all [l10n-central](https://hg.mozilla.org/l10n-central) repositories.
-* Clone [gecko-strings-quarantine](https://hg.mozilla.org/l10n/gecko-strings-quarantine/) and [gecko-strings](https://hg.mozilla.org/l10n/gecko-strings).
+* Clone the [firefox-l10n](https://github.com/mozilla-l10n/firefox-l10n) repository.
+* Clone the [firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source) repository.
 
-A set of scripts to automate cloning and updating of the l10n-central repositories is [available here](https://github.com/flodolo/scripts/tree/master/mozilla_l10n/clone_hgmo).
+### Ensure there are no blocking issues in the update branch of firefox-l10n-source
 
-### Ensure there are no blocking issues in gecko-strings-quarantine
-
-As part of migrations, content needs to be pushed from the quarantine repository to `gecko-strings`. Since content can’t be pushed selectively, any pending issue needs to be solved before starting the process (eventually backing out problematic changes). For more information about the review process, see [this document](review.md).
+As part of migrations, content needs to be pushed from the `update` branch to the `main` branch of `firefox-l10n-source`. Since content can’t be pushed selectively, any pending issue needs to be solved before starting the process (eventually backing out problematic changes). For more information about the review process, see [this document](review.md).
```

Comment:
This is not true at this point. You can make manual changes to the `update` branch to take out issues.